src/pages/streaming/streaming.html

```diff
@@ -0,0 +1,16 @@
+<ion-header>
+  
+  <ion-navbar>
+    <ion-title>TV</ion-title>
+  </ion-navbar>
+  
+</ion-header>
+<ion-content padding>
+  <div class="video-container">
+    <video #player controls width="640" height="360" style="margin: auto;">
+      <source [src]="url" type="video/mp4">
+      <source [src]="url" type='video/webm;codecs="vp8, vorbis"' />
+      <source [src]="url" type='video/mp4;codecs="avc1.42E01E, mp4a.40.2"' />  
```

Comment:
how did you test these 3 alternatives ? does it exist more ?

Reply:
I've tested on ios, android and desktop, and this solution it worked on this contexts

Reply:
I've reviewed and just add  video/mp4 source it worked